src/main/java/calculator/util/Validator.java

```diff
@@ -0,0 +1,93 @@
+package calculator.util;
+
+import static calculator.message.Delimiter.CUSTOM_DELIMITER;
+import static calculator.message.ErrorMessage.CUSTOM_DELIMITER_CANNOT_BE_NUMBER_MESSAGE;
+import static calculator.message.ErrorMessage.EMPTY_ELEMENT_MESSAGE;
+import static calculator.message.ErrorMessage.EMPTY_INPUT_MESSAGE;
+import static calculator.message.ErrorMessage.INVALID_ELEMENT_MESSAGE;
+import static calculator.message.ErrorMessage.INVALID_FLOAT_INPUT_MESSAGE;
+import static calculator.message.ErrorMessage.NEGATIVE_NUMBER_MESSAGE;
+
+import java.util.List;
+
+public class Validator {
+    public static String validateInput(String userInput) {
+        isBlank(userInput);
+        String newInput = validateCustomDelimiter(userInput);
+
+        return newInput;
+    }
+
+    public static void isNegativeNumber(List<Integer> numbers) {
+        numbers.forEach(number -> {
+            if (number < 0) {
+                throw new IllegalArgumentException(NEGATIVE_NUMBER_MESSAGE);
+            }
+        });
+    }
```

Comment:
```suggestion
    public static void validateContainsNegativeNumber(List<Integer> numbers) {
        numbers.forEach(number -> {
            if (number < 0) {
                throw new IllegalArgumentException(NEGATIVE_NUMBER_MESSAGE);
            }
        });
    }
```

인자가 여러 값이라서 `is`보다는 `contains`가 적절해 보입니다!
또한 메서드명이 `is`라서 `boolean`이 반환될 것으로 예상했는데, void였네요!
메서드 네이밍의 보편적인 기준을 찾아보셔도 좋을 것 같습니다 😄